### ISSUE TITLE
Plans: plan detail header should be one accessibility element and a description would be helpful

### ISSUE THREAD

**Issue by designsimply** [wordpress-mobile/WordPress-iOS#12855]
Steps to reproduce:

1. Go to Settings > General > Accessibility > VoiceOver and toggle the setting on.
1. In the WordPress app, go to Plans and tap on any plan to go to the details screen.
1. Swipe right to access the header and browse through the screen.

**Result:** the plan detail header is not grouped, and a description would be helpful (i.e: “Features on this plan”…). ([3s](https://cloudup.com/cW2AUBnzKk9))

<img src="https://user-images.githubusercontent.com/1119271/68171191-0b068300-ff30-11e9-9fb2-4fe3f82c80f7.png" width="270" alt="voiceover-plan-detail-header-is-not-grouped">
<sup>Tested with WPiOS 13.5.0.3 (App Store version) on iPhone 6S iOS 13.2.</sup>

<p style="clear:both;"><sub>/hat tip @etoledom for the report, internal reference: p77Llu-bQ6-p2</sub></p>

**Comment by dangermattic** [comment]
Thanks for reporting! 👍
